src/psyclone/parse/__init__.py

```diff
@@ -31,3 +31,18 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 # -----------------------------------------------------------------------------
+# Author: J. Henrichs, Bureau of Meteorology
+
+'''This directory contains classes related to parsing Fortran.
+'''
+
+from psyclone.parse.module_info import ModuleInfo, ModuleInfoError
+from psyclone.parse.module_manager import ModuleManager
+
+# The entities in the __all__ list are made available to import directly from
```

Comment:
This isn't true (but we used to think it was). The only effect of setting `__all__` for our purposes is the AutoAPI documentation generation.

Reply:
Fixed, thanks. I forgot that (and apparently used the wrong file as template :) )